workshops/migrations/0060_published_help_text.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+import re
+import django_countries.fields
+
+
+def add_address_to_online_events(apps, schema_editor):
+    """Set Event.address if empty for online events."""
+    Event = apps.get_model('workshops', 'Event')
+    Tag = apps.get_model('workshops', 'Tag')
+    online = Tag.objects.get(name='online')  # should be created via 0040_*
+
+    Event.objects.filter(tags__in=[online]) \
+                 .filter(models.Q(address=None) | models.Q(address='')) \
+                 .update(address='Internet')
```

Comment:
Are we still keeping an online tag in parallel with the online country / address / lat / lon?  That doesn't seem very DRY :p.  If we need the online country / address / lat / long for the (un)published logic, maybe we can drop the online tag?  In any case, this migration should also be setting the country, lat, and lon.  And you'll want another update call that gets everything with the W3 country and forces the address, lat, and lon to the values from 0040.  I wouldn't bother about the `Q` filters in either update query, because a bit of extra clobbering isn't a big deal for a one-off migration, and it ensures we don't miss some logic and leave an online event with the wrong values.